Sources/StreamChat/Controllers/SearchControllers/UserSearchController/UserSearchController.swift

```diff
@@ -22,72 +22,29 @@ public class ChatUserSearchController: DataController, DelegateCallable, DataSto
     /// The `ChatClient` instance this controller belongs to.
     public let client: ChatClient
     
-    /// Filter hash this controller observes.
-    let explicitFilterHash = UUID().uuidString
-    
-    lazy var query: UserListQuery = {
-        // Filter is just a mock, explicit hash will override it
-        var query = UserListQuery(filter: .exists(.id), sort: [.init(key: .name, isAscending: true)])
-        // Setting `shouldBeObserved` to false prevents NewUserQueryUpdater to pick this query up
-        query.shouldBeUpdatedInBackground = false
-        // The initial DB fetch will return 0 users and this is expected
-        // In the future we'll implement DB search too
-        query.filter?.explicitHash = explicitFilterHash
-        
-        return query
-    }()
-    
     /// Copy of last search query made, used for getting next page.
-    var lastQuery: UserListQuery?
+    public private(set) var query: UserListQuery?
     
-    /// The users matching the query of this controller.
-    ///
-    /// To observe changes of the users, set your class as a delegate of this controller or use the provided
-    /// `Combine` publishers.
-    ///
-    public var users: LazyCachedMapCollection<ChatUser> {
-        startUserListObserverIfNeeded()
-        return userListObserver.items
```

Comment:
This is a breaking change 🔴  Why are we removing this one? We need at least to deprecate it, not remove it completely.

Reply:
I moved the deprecation back to this file, as agreed.